reporter/src/funTest/kotlin/reporters/ExcelReporterFunTest.kt

```diff
@@ -26,6 +26,9 @@ import io.kotest.matchers.shouldBe
 
 import java.io.File
 
+import org.apache.poi.ss.usermodel.Cell
```

Comment:
I don't understand the commit message, what is the actual problem solved by this change? Is it just a preparation for the next commit? How is resaving the file related to this?

Reply:
So, the change is complete unrelated to the next commit; it's just a preparation to be able to conveniently update `file-counter-expected-scan-report.xlsx` with Excel, if we ever need to do so.

Reply:
Thanks for the explanation.